pkg/datapath/iptables/iptables.go

```diff
@@ -598,7 +600,7 @@ func (m *Manager) installStaticProxyRules() error {
 		if err := ip4tables.runProg([]string{
 			"-t", "filter",
 			"-A", ciliumOutputChain,
-			"-m", "mark", "--mark", matchProxyReply,
+			"-m", "mark", "--mark", matchFromProxy,
 			"-m", "comment", "--comment", "cilium: ACCEPT for proxy return traffic",
```

Comment:
Just a cosmetic update:

```suggestion
			"-m", "comment", "--comment", "cilium: ACCEPT for proxy traffic",
```

there's also a comment on top that needs `s/proxy return/proxy`.

Same for for IPv6 below :).